Tails specific: we are amnesic, no need to save mixer levels on shutdown.

--- chroot.orig/etc/init.d/alsa-utils	2012-09-24 10:05:12.749039812 +0200
+++ chroot/etc/init.d/alsa-utils	2012-09-24 10:47:23.717869294 +0200
@@ -10,1 +10,1 @@
-# Default-Stop:      0 1 6
+# Default-Stop:
